app/assets/javascripts/app/hms-formatter.js

```diff
@@ -0,0 +1,26 @@
+const root = window.LoginGov = (window.LoginGov || {});
+
+function pluralize(word, count) {
+  return `${word}${count !== 1 ? 's' : ''}`;
+}
+
+function formatMinutes(minutes) {
+  if (!minutes) return 0;
+
+  return `${minutes} ${pluralize('minute', minutes)}`;
+}
+
+function formatSeconds(seconds) {
+  return `${seconds} ${pluralize('second', seconds)}`;
+}
+
+root.hmsFormatter = function(milliseconds) {
```

Comment:
why do we want these methods attached to our LoginGov object (vs. just a function that we export / import within the js modules that need it)?